README.md

```diff
@@ -30,7 +30,7 @@ This repository is a monorepo and contains a collection of React Native modules
 > [!IMPORTANT]  
 > The Adobe React Native SDK utilizes the React Native interop layer to ensure compatibility with new architecture.
 
-## Requirements
+## Requirements 
```

Comment:
For `iOS Privacy Manifest` section, let change small update:

Starting from Adobe Experience Platform React Native 6.x, the libraries depend on Experience Platform iOS 5.x SDKs, which have been updated to align with Apple's latest guidelines on [privacy manifest](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files). For further details on how Apple's privacy-related announcements affect the Adobe mobile SDK for iOS, please refer to this [document](https://developer.adobe.com/client-sdks/resources/privacy-manifest/).